spec/integration/report_spec.rb

```diff
@@ -4,7 +4,7 @@
   before :each do
     webauth = double('WebAuth', :login => 'sunetid', :attributes => {'DISPLAYNAME' => 'Example User'}, :privgroup => ADMIN_GROUPS.first)
     @current_user = User.find_or_create_by_webauth(webauth)
-    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@current_user)
+    allow_any_instance_of(CatalogController).to receive(:current_user).and_return(@current_user)
```

Comment:
Just tried with the following:
 - `ActionController::Base` - :x: message not received
 - `Argo::Application::ApplicationController` :x: message not received